src/views/Register/Inner.jsx

```diff
@@ -0,0 +1,125 @@
+import { Button, Form } from 'antd';
+import AccountLayout from 'layouts/Account';
+import { memo } from 'react';
+import { NavLink } from 'react-router-dom';
+import routeConstants from 'route/routeConstant';
+import 'layouts/Account/index.scss';
+import AccountInput from 'layouts/Account/AccountInput';
+import AutopostLogo48 from 'assets/images/AutopostLogo48';
+
+const Inner = memo(({ handleRegister }) => {
+    //Validate rules
+    const nameRules = [
+        {
+            pattern: /^[^\d]+$/,
```

Comment:
ai quy định cái rule này vậy

Reply:
Này bên BE gửi e hồi mới làm project, để e update lại ạ